packages/generator-emakinacee-react/generators/app/templates/shared/dot/.eslintrc

```diff
@@ -1,7 +1,10 @@
 {
     "extends": [
         "emakinacee-base",
-        "emakinacee-react"
+        "emakinacee-react",
+        // Must be last to override rules. https://github.com/prettier/eslint-config-prettier#installation
+        "prettier",
+        "prettier/react"
```

Comment:
@dgoerdes Have you created the issue? I am just asking, if I understood the goal correctly: _Remove all rules which would be overwritten by the prettier config?_